### PR TITLE
rtlutility: Add version 1.0.3

### PR DESCRIPTION
**Pull request by moshen**
Add RTL Utility 1.0.3 from Oblique Audio https://oblique-audio.com/rtl-utility.php

Please let me know if there's a better way to do the `pre` and `post` scripts to make the shortcuts work.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/rtlutility.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "version": "1.0.3",
+    "description": "RTL Utility is a tool for measuring the Round Trip Latency of your Digital Audio Workstation (DAW) and audio interface",
+    "homepage": "https://oblique-audio.com/rtl-utility.php",
+    "license": "Freeware",
+    "suggest": {
+        "vcredist2015": "extras/vcredist2015",
+        "vcredist2017": "extras/vcredist2017",
+        "vcredist2019": "extras/vcredist2019"
```
Comment (tech189) [inline bucket/rtlutility.json:9]:
```suggestion
    "suggest": {
        "Microsoft Visual C++ Redistributables": "extras/vcredist2022"
```
The latest version includes all the redistributables for versions 2015-2019 - see https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022

File: bucket/rtlutility.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_x64.exe",
+            "hash": "5764c78d4e7150d187dfd9312b21a77c4d4cbecf54586f83cea7f83ae6918a2b"
+        },
+        "32bit": {
+            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_Win32.exe",
```
Comment (tech189) [inline bucket/rtlutility.json:17]:
```suggestion
        "64bit": {
            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_x64.exe#/RTL_Utility.exe",
            "hash": "5764c78d4e7150d187dfd9312b21a77c4d4cbecf54586f83cea7f83ae6918a2b"
        },
        "32bit": {
            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_Win32.exe#/RTL_Utility.exe",
```
We can change the name of the downloaded file with this syntax, avoids any renaming in pre_/post_install!

File: bucket/rtlutility.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        }
+    },
+    "pre_install": "New-Item \"${dir}/RTL_Utility.exe\" | Out-Null",
+    "post_install": [
+        "pushd $dir",
+        "rm RTL_Utility.exe",
+        "dir .\\RTL_Utility_*.exe | Rename-Item -NewName \"RTL_Utility.exe\"",
+        "popd"
+    ],
+    "shortcuts": [
```
Comment (tech189) [inline bucket/rtlutility.json:28]:
```suggestion
    },
    "shortcuts": [
```
Remove this, as files are renamed by scoop above in the url section

File: bucket/rtlutility.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        "re": "RTL_Utility_(?<version>(\\d+)_(\\d+)_(\\d+))_x64\\.exe",
+        "replace": "${1}.${2}.${3}"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://oblique-audio.com/downloads/RTL_Utility_$matchVersion_x64.exe"
+            },
+            "32bit": {
+                "url": "https://oblique-audio.com/downloads/RTL_Utility_$matchVersion_Win32.exe"
```
Comment (tech189) [inline bucket/rtlutility.json:45]:
```suggestion
            "64bit": {
                "url": "https://oblique-audio.com/downloads/RTL_Utility_$matchVersion_x64.exe#/RTL_Utility.exe"
            },
            "32bit": {
                "url": "https://oblique-audio.com/downloads/RTL_Utility_$matchVersion_Win32.exe#/RTL_Utility.exe"
```

Same syntax as in url, need it here too for further updates

**Review by tech189 — COMMENTED** [review]
I made some suggestions to simplify the manifest, now you can avoid pre_/post_install scripts by using the `#/abc.exe` renaming syntax.

File: bucket/rtlutility.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
+{
+    "version": "1.0.3",
+    "description": "RTL Utility is a tool for measuring the Round Trip Latency of your Digital Audio Workstation (DAW) and audio interface",
```
Comment (niheaven) [inline bucket/rtlutility.json:3]:
```suggestion
    "description": "Tool for measuring the Round Trip Latency of your Digital Audio Workstation (DAW) and audio interface",
```

**Comment by moshen** [comment]
@tech189 Thanks! That's what I was looking for

**Comment by moshen** [comment]
@niheaven I just copied the first line of their own description of the utility. Your change sounds better

File: bucket/rtlutility.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    "shortcuts": [
+        [
+            "RTL_Utility.exe",
+            "RTL Utility"
+        ]
+    ],
+    "checkver": {
+        "url": "https://oblique-audio.com/rtl-utility.php",
+        "re": "RTL_Utility_(?<version>(\\d+)_(\\d+)_(\\d+))_x64\\.exe",
+        "replace": "${1}.${2}.${3}"
```
Comment (niheaven) [inline bucket/rtlutility.json:28]:
```suggestion
        "regex": "<td>([\\d.]+)</td>"
```

File: bucket/rtlutility.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://oblique-audio.com/downloads/RTL_Utility_$matchVersion_x64.exe#/RTL_Utility.exe"
```
Comment (niheaven) [inline bucket/rtlutility.json:33]:
```suggestion
                "url": "https://oblique-audio.com/downloads/RTL_Utility_$underscoreVersion_x64.exe#/RTL_Utility.exe"
```

Comment (moshen) [reply bucket/rtlutility.json:33]: Ahh, totally missed this on  https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#version-variables

**Comment by moshen** [comment]
@niheaven Added your changes, but it still says "Changes Requested". Maybe I'm missing something.

**Review by niheaven — APPROVED** [review]

**Pull request merged**